openapi-generator/lib/src/models/command.dart

```diff
@@ -21,11 +21,14 @@ class Command {
     required String executable,
     required List<String> arguments,
   }) : this._(
-          executable == 'dart' || wrapper == Wrapper.none
+          wrapper == Wrapper.none
```

Comment:
Why was this removed? This should short-circuit in the event that the dart command is being used. It is possible for the executable to be either a plain `dart` or `flutter` call 🤔 . We likely want to prevent configurations that have a wrapper applied to the dart command. TBH this could be clean up a bit to not have nested turneries.